### ISSUE TITLE
add to docs.stacks.co

### ISSUE THREAD

**Issue by wileyj** [Trust-Machines/stacks-dev-guide#1]
could this doc be pr'ed to <https://github.com/stacks-network/docs> ?

i'm not sure where or what updates would be needed offhand, but it's great content i think should be on the docs

**Comment by igorsyl** [comment]
Yes! We'll move to the docs repo once this doc is more complete. We are using this repo because we don't need any PR approvals. @wileyj
